components/script_bindings/webidls/QuotaExceededError.webidl

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+// https://heycam.github.io/webidl/#quotaexceedederror
+
+[Exposed=(Window,Worker,Worklet,DissimilarOriginWindow), Serializable]
```

Comment:
https://github.com/servo/servo/pull/38615 means the build will fail, since this is marked Serializable but we haven't implemented the actual serialization (eg https://github.com/servo/servo/pull/36535). I'm fine with filing a separate followup issue for that work, but we will need to comment out this annotation in the meantime.